### PR TITLE
feat: allow multiple KMS keys to create CMEK database/backup

### PR DESCRIPTION
**Pull request by hwin16**
Add kms_key_names field to create database/create backup code snippets.

### CODE REVIEW

File: protos/protos.d.ts [hunk 1]
```diff
@@ -18436,9 +18436,6 @@ export namespace google {
                     /** ReadAction limit. */
                     public limit: number;
 
-                    /** ReadAction _index. */
```
Comment (surbhigarg92) [inline protos/protos.d.ts:18439]: proto changes should be autogenerated and should not be part of this PR.

File: samples/backups-create-with-multiple-kms-keys.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+/**
```
Comment (surbhigarg92) [inline samples/backups-create-with-multiple-kms-keys.js:1]: Sample content is all good but new samples should follow [rubric](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head#format-overview) format . Refer sample https://github.com/googleapis/nodejs-spanner/blob/main/samples/instance-partition-create.js

**Comment by surbhigarg92** [comment]
Samples are missing Integration tests . Please add corresponding integration test for these samples.

File: samples/backups-create-with-multiple-kms-keys.js (hunk 2 above)
Comment (hwin16) [reply samples/backups-create-with-multiple-kms-keys.js:1]: Done. PTAL.

**Comment by hwin16** [comment]
@surbhigarg92 , Added integration tests. I have ran my test using 'span-cloud-testing' project and it passed. When you have approved the merge request and kokoro presubmit uses a different project, test cases will fail. Then, I will leave a todo and skip the test. 

Screenshot 2024-09-07 at 11 46 23 PM

**Review by surbhigarg92 — APPROVED** [review]

**Comment by harshachinta** [comment]
Had an offline discussion with @hwin16 . We will remove skipping tests and the PR will fail.
We will merge this once the backend is ready.
